packages/cozy-clisk/src/contentscript/ContentScript.js

```diff
@@ -235,7 +237,12 @@ export default class ContentScript {
     this.onlyIn(WORKER_TYPE, 'waitForElementNoReload')
     log.debug('waitForElementNoReload', selector)
     await waitFor(() => Boolean(document.querySelector(selector)), {
-      timeout: DEFAULT_WAIT_FOR_ELEMENT_TIMEOUT
+      timeout: {
+        milliseconds: DEFAULT_WAIT_FOR_ELEMENT_TIMEOUT,
+        message: new TimeoutError(
+          `waitForElementNoReload ${selector} timed out after ${DEFAULT_WAIT_FOR_ELEMENT_TIMEOUT}ms`
+        )
+      }
     })
     return true
   }
```

Comment:
Nice, but how this error can't be converted to a job error and stop the execution of the konnector?

Reply:
@Crash-- It would be nice to merge this PR to have better timeout error messages, even if it does not fix any problem. Can we merge it ?